.gitignore

```diff
@@ -10,3 +10,6 @@ html/
 
 # automatically generated crates
 /stm32*/
+
+# MacOS
+.DS_Store
```

Comment:
I don't think platform- and tool-specific ignores should go in repository `.gitignores`; otherwise we should start adding ignores for vim temporary files, Windows thumbnail databases, MacOS's `._`, and so on. Would it work for you to put `.DS_Store` in your global git ignore (`.config/git/ignore`)?

Reply:
yes sure, makes sense